content/millennium/r4/proprietary-codes-and-systems.md

```diff
@@ -162,7 +162,8 @@ The Patient Resource supports proprietary codes for:
 The RelatedPerson Resource supports proprietary codes for:
 
 * RelatedPerson.communication.language codes are maintained in [Code Set 36 Languages](#code-set-36-languages)
-* RelatedPerson.relationship codes are maintained in [Code Set 40 Person Relationship Types](#code-set-40-person-relationships) and [Code Set 351 Person Relationship Types](#code-set-351-person-relationship-types)
+* RelatedPerson.relationship codes are maintained in [Code Set 351 Person Relationship Types](#code-set-351-person-relationship-types)
+* The Relation extensions on RelatedPerson.relationship are maintained in [Code Set 40 Person Relationship Types](#code-set-40-person-relationships)
```

Comment:
Should we say "The Relation extension codes on RelatedPerson.relationship are maintained in....."?

Reply:
changed here https://github.com/cerner/fhir.cerner.com/pull/644/commits/1252981d25f41e6d89385eade6f91090cdce9aff#diff-aedba7f37b773d7a314400db492774b8a9d406ed43aad2101793b82970106103R166

Reply:
+1 on new phrasing.